### PR TITLE
Update examples in document

### PR DESCRIPTION
**Pull request by djKooks**
Hello,
I update document to add examples for other protocols. Currently there are only MySQL, but planning to add some more later.
Also I changed form a bit as @naoyak suggested.

Could you look on it?

### CODE REVIEW

**Comment by matthewwardrop** [comment]
Hi @djKooks!

Thanks for your patch(es) and for continuing to improve our documentation :). I really appreciate it!

One of the key tenets of `omniduct` was that all ducts of the same "kind" behaved essentially indistinguishably (perhaps with a few minor quirks) to each another, and so if we truly require multiple examples in our documentation for each protocol, we would mask that simplicity and/or somehow this targetted simplicity is being violated. The only significant differences between ducts of the same kind should be in the constructor, for which the parameters are all documented in `help(<duct class>)`.

Currently, `presto` is used as an example, but IIRC, I noted that every database duct behaves almost identically. I also made sure that the full API documentation for each `Duct` subclass was present in the generated docs.

With all that in mind, can you give some feedback as to what you feel the existing documentation was lacking? For example, was the problem that you couldn't copy and paste code for a super-quick-start? Depending on your response, perhaps instead adding a whole host of (essentially identical) examples we can instead just explicitly link to the constructor API? Or if the consensus is that this is too hard to follow, perhaps limit the duplication of code by just providing examples for different protocols of the construction of Duct instances? But even that is usually straightforward and exactly the same, since: <DuctSubclass>(host=<blah>, port=<port>, username=<>, password=<>) covers 99% of cases.

Curious on your thoughts too, @naoyak @danfrankj

**Comment by naoyak** [comment]
Yeah, I should have thought about this more before @djKooks started this PR. If anything maybe we should have an example highlighting how easy it is to switching between different Duct sources. It depends on the use case, really.

**Comment by djKooks** [comment]
@matthewwardrop @naoyak  thanks for comment!
I think your suggestion is correct. Than how about to divide example documents with the one using Duct(and comment it runs by same code with all protocols), and one using each protocol client? 

Like:
```
Quickstart
|- Example: Duct
|- Example: Protocol Client
    |- Presto
    |- MySQL
    ...
```

**Comment by naoyak** [comment]
@djKooks I was thinking more along the lines of having e.g. one code example for `Duct` which shows the class being initialized for several different protocols. The concern was that breaking Presto, MySQL, Hive, etc...into separate pages or examples is too fragmented and draws attention away from the common API.

**Comment by djKooks** [comment]
@naoyak this is just a question...
If the purpose of this module is to simplify the connection of variable protocols, is client module for each protocol(PrestoClient, SQLAlchemyClient, etc) needed?

**Comment by matthewwardrop** [comment]
Hi @djKooks; yes, a separate subclass is required for each client in order to be a wrapper that implements the common API for a given protocol. However, once implemented, each client should function more-or-less identically. The differences and quirks for each client are clearly documented, and the API is enumerated with this documentation (e.g. https://omniduct.readthedocs.io/en/latest/api/databases/overview.html#subclass-reference). As such, I'm still a little unsure what is lacking in the documentation. Is it that the description in the quickstart page of the documentation is not clear about where to find documentation for each client?

**Comment by djKooks** [comment]
@matthewwardrop okay. Actually, it is not quite complicate to find out the implementing way of various protocol by using `Duct`, but I just wanted to show the way more clearly.

- Than, in `Duct` example, how about give a comment more clearly(I'll think of the way...) that other protocols will work same?
- I'll think more about examples for subclasses...
